packages/rn-tester/js/examples/TurboModule/NativeCxxModuleExampleExample.js

```diff
@@ -155,9 +203,6 @@ class NativeCxxModuleExampleExample extends React.Component<{||}, State> {
         'Cannot load this example because TurboModule is not configured.',
       );
     }
-    Object.keys(this._tests).forEach(item =>
-      this._setResult(item, this._tests[item]()),
-    );
```

Comment:
Can we do this for all platforms now?

Reply:
I'm not sure what you mean.

I've removed the autorun of the tests because some of the crash tests can't be watched in JS at the moment and the app would crash immediately.